demo/web/all_demos.dart

```diff
@@ -0,0 +1,404 @@
+import 'iron_pages/iron_pages_demo.dart';
+import 'iron_range_behavior/iron_range_behavior_demo.dart';
+import 'iron_resizable_behavior/iron_resizable_behavior_demo.dart';
+import 'iron_selector/iron_selector_demo.dart';
+import 'iron_signals/iron_signals_demo.dart';
+import 'iron_validatable_behavior/iron_validatable_behavior_demo.dart';
+import 'iron_validator_behavior/iron_validator_behavior_demo.dart';
+import 'marked_element/marked_element_demo.dart';
+import 'paper_badge/paper_badge_demo.dart';
+//import 'paper_behaviors/paper_behaviors_demo.dart';
+import 'paper_button/paper_button_demo.dart';
+import 'paper_card/paper_card_demo.dart';
+import 'paper_checkbox/paper_checkbox_demo.dart';
+import 'paper_dialog/paper_dialog_demo.dart';
+import 'paper_dialog_behavior/paper_dialog_behavior_demo.dart';
+import 'paper_dialog_scrollable/paper_dialog_scrollable_demo.dart';
+import 'paper_drawer_panel/paper_drawer_panel_demo.dart';
+import 'paper_dropdown_menu/paper_dropdown_menu_demo.dart';
+import 'paper_fab/paper_fab_demo.dart';
+import 'paper_header_panel/paper_header_panel_demo.dart';
+import 'paper_icon_button/paper_icon_button_demo.dart';
+import 'paper_input/paper_input_demo.dart';
+import 'paper_item/paper_item_demo.dart';
+//import 'paper_listbox/paper_listbox_demo.dart';
+import 'paper_material/paper_material_demo.dart';
+import 'paper_menu/paper_menu_demo.dart';
+import 'paper_menu_button/paper_menu_button_demo.dart';
+import 'paper_progress/paper_progress_demo.dart';
+import 'paper_radio_button/paper_radio_button_demo.dart';
+import 'paper_radio_group/paper_radio_group_demo.dart';
+import 'paper_ripple/paper_ripple_demo.dart';
+import 'paper_scroll_header_panel/paper_scroll_header_panel_demo.dart';
+import 'paper_slider/paper_slider_demo.dart';
+import 'paper_spinner/paper_spinner_demo.dart';
+import 'paper_styles/paper_styles_demo.dart';
+import 'paper_tabs/paper_tabs_demo.dart';
+import 'paper_toast/paper_toast_demo.dart';
+import 'paper_toggle_button/paper_toggle_button_demo.dart';
+import 'paper_toolbar/paper_toolbar_demo.dart';
+import 'paper_tooltip/paper_tooltip_demo.dart';
+//  import 'chartjs_element/index.dart';
+//  import 'google_domain_user_picker/google_domain_user_picker_demo.dart';
+//  import 'google_drive/google_drive_demo.dart';
+//  import 'google_recaptcha/google_recaptcha_demo.dart';
+// import 'iron_label/iron_label_demo.dart';
+//  import 'neon_animation/card/index.dart';
+//  import 'neon_animation/declarative/index.dart';
+//  import 'neon_animation/doc/index.dart';
+//  import 'neon_animation/dropdown/index.dart';
+//  import 'neon_animation/grid/index.dart';
+//  import 'neon_animation/list/index.dart';
+//  import 'neon_animation/load/index.dart';
+//  import 'neon_animation/reprojection/index.dart';
+//  import 'neon_animation/tiles/index.dart';
+
+
+/// Silence analyzer [DemoElements], [PaperItem], [PaperDrawerPanel],
+/// [PaperHeaderPanel], [IronList], [PaperToolbar],
```

Comment:
probably makes sense to just condense all of these onto as few lines as possible